src/latt-gui/lattmainwindow.cpp

```
@@ -397,7 +399,7 @@ void lattMainWindow::updateInput() {
 }
 
 void lattMainWindow::updateOutput() {
-    QFileInfo fi = lattui.lineInput->text();
+    QFileInfo fi(lattui.lineInput->text());
```

Clang (14.0.6)

[warning] variable 'fi' is not initialized

(cppcoreguidelines-init-variables)